SmartDeviceLink/public/SDLAudioData.m

```diff
@@ -0,0 +1,122 @@
+//
+//  SDLAudioData.m
+//  SmartDeviceLink
+//
+//  Created by Nicole on 11/9/20.
+//  Copyright © 2020 smartdevicelink. All rights reserved.
+//
+
+#import "SDLAudioData.h"
+
+#import "SDLFile.h"
+#import "SDLSpeak.h"
+#import "SDLTTSChunk.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLAudioData()
+
+/// The audio file data that will be uploaded.
+@property (nullable, copy, nonatomic, readonly) NSDictionary<NSString *, SDLFile *> *audioFileData;
```

Comment:
This can just be a mutable dictionary to simplify some code below in `addAudioFiles`